app/src/androidTest/java/at/tugraz/ist/swe/cheatapp/testTextField.java

```diff
@@ -0,0 +1,38 @@
+package at.tugraz.ist.swe.cheatapp;
+
+import android.support.test.rule.ActivityTestRule;
+import android.support.test.runner.AndroidJUnit4;
+import android.widget.EditText;
+
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import static android.support.test.espresso.Espresso.onView;
+import static android.support.test.espresso.action.ViewActions.closeSoftKeyboard;
+import static android.support.test.espresso.action.ViewActions.replaceText;
+import static android.support.test.espresso.action.ViewActions.typeText;
+import static android.support.test.espresso.assertion.ViewAssertions.matches;
+import static android.support.test.espresso.matcher.ViewMatchers.isDisplayed;
+import static android.support.test.espresso.matcher.ViewMatchers.withId;
+import static android.support.test.espresso.matcher.ViewMatchers.withText;
+
+@RunWith(AndroidJUnit4.class)
+public class testTextField {
+
+    @Rule
+    public ActivityTestRule<MainActivity> mainActivityTestRule = new ActivityTestRule<>(MainActivity.class);
+
+    @Test
+    public void testFieldVisible() {
+        onView(withId(R.id.textEntry)).check(matches(isDisplayed()));
+    }
+
+    @Test
+    public void testInput(){
+        String test_text = "Test_Input";
```

Comment:
Please use camel-case naming scheme (e.g. testText for methods/members/variables TestClass for class names)